journald/journald.cpp

```diff
@@ -335,6 +412,9 @@ class JournaldLoggerProcess : public Process<JournaldLoggerProcess>
   int num_entries;
   struct iovec* entries;
 
+  // The active connection to the specified fluentbit address.
```

Comment:
Why "active"?

Reply:
Hmm... I guess the connection could die.  I'll remove the adjective.